exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<p>I wouldn't use this wrapper here. It's another step to explain, another "extractor something". I think it's better to have a piece of code that can be copy/pasted to the user'program. If this ExtractionWrapper is universally useful then why is not in the Text AI?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>for the moment AI-Lab</p>